src/test/integration/IntegrationChecks.t.sol

```diff
@@ -127,4 +127,10 @@ contract IntegrationCheckUtils is IntegrationBase {
         assert_Snap_Unchanged_OperatorShares(operator, "operator should have shares unchanged");
         assert_Snap_Unchanged_StrategyShares(strategies, "strategies should have total shares unchanged");
     }
+
+    function assertRegisteredAsOperatorState(User staker) internal {
```

Comment:
I'd argue we don't need a function for this since it's only a one line check - cleaner to just keep it in original test file